src/main/java/hwr/oop/riddler/logic/backtrackingsolver/component/PossiblesEliminator.java

```diff
@@ -0,0 +1,28 @@
+package hwr.oop.riddler.logic.backtrackingsolver.component;
+
+import hwr.oop.riddler.model.Sudoku;
+import hwr.oop.riddler.model.component.Cell;
+import hwr.oop.riddler.model.component.CellGroup;
+
+public class PossiblesEliminator implements SolvingComponent {
+    boolean changesWereMade;
+
+    @Override
+    public boolean execute(Sudoku sudoku) {
+        changesWereMade = false;
+
+        for (CellGroup cellGroup : sudoku.getConcatenatedCellGroups()) {
+            removePossibleCellValues(cellGroup);
+        }
+
+        return changesWereMade;
```

Comment:
ich würde this.changesWereMade damit man nicht drüber stolpert dass die value durch die removePossibleCellValues changesWereMade geändert wird